docs/api/files.md

```diff
@@ -0,0 +1,77 @@
+# Files
+
+> This page is incomplete, stay tuned.
+
+## Upload file
+
+### Code samples
+
+```shell
+# Example request using curl
+curl -X POST http://coder-server:8080/api/v2/files \
+  -H 'Accept: application/json' \
+  -H 'Content-Type: application/x-tar' \
+  -H 'Coder-Session-Token: API_KEY'
+```
+
+`POST /files`
+
+Notice: Swagger 2.0 doesn't support file upload with a `content-type` different than `application/x-www-form-urlencoded`.
+
+> Body parameter
+
+```yaml
+file: string
+```
+
+### Parameters
+
+| Name           | In     | Type           | Required | Description                              |
+| -------------- | ------ | -------------- | -------- | ---------------------------------------- |
+| `Content-Type` | header | string         | true     | Content-Type must be `application/x-tar` |
+| `body`         | body   | object         | true     |                                          |
+| `» file`       | body   | string(binary) | true     | File to be uploaded                      |
```

Comment:
Is this actually a string? Is it base64 encoded? If not, perhaps calling this type `data` or `binary` would be easier to understand?

Reply:
Renamed!